src/mixxx.cpp

```diff
@@ -451,7 +469,15 @@ void MixxxMainWindow::finalize() {
     Timer t("MixxxMainWindow::~finalize");
     t.start();
 
-    // Save the current window state (position, maximized, etc)
+    UserSettingsPointer pConfig = m_pSettingsManager->settings();
+    int inhibit = pConfig->getValue<int>(ConfigKey("[Config]","InhibitScreensaver"));
+    mixxx::ScreenSaverPreference inhiPref = mixxx::ScreenSaverPreference(inhibit);
```

Comment:
use of "auto" and explicit cast sounds interesting. use of static_cast<int> sounds in contradiction with the use of "enum class".
I would preffer the oppinion of someone else before changing that from what it is now.